File: web-api-doc/src/main/java/net/yan100/compose/webapidoc/autoconfig/AutoConfigEntrance.java
package net.yan100.compose.webapidoc.autoconfig;

import net.yan100.compose.webapidoc.properties.SwaggerProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Import;

@Import({
  OpenApiDocConfig.class,
  // 暂时不需要了
  //OpenApiConfigController.class
})
@EnableConfigurationProperties(SwaggerProperties.class)
public class AutoConfigEntrance {
}

File: shared/src/main/kotlin/net/yan100/compose/autoconfig/AutoConfigEntrance.kt
package net.yan100.compose.autoconfig

import net.yan100.compose.properties.DataLoadProperties
import net.yan100.compose.properties.SnowflakeProperties
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.ComponentScan

@EnableConfigurationProperties(SnowflakeProperties::class, DataLoadProperties::class) @ComponentScan("net.yan100.compose.autoconfig") class AutoConfigEntrance
